### DISCUSSION TITLE
Best image for the PGA2040

### DISCUSSION THREAD

**Discussion by bencaldwell78** [OpenStickCommunity/GP2040-CE#992] [Q&A]
Has anyone used the PiMoroni PGA2040? If so, which GP2040-CE image did you use?

The PGA2040 is a tiny and basic RP2040 breakout, with minimal bells and whistles. I was curious what GP2040-CE image would work best with it.

**Comment by TheTrainGoes** [comment]
Hi @bencaldwell78 ,

Since the PGA2040 breaks out all of the pins you can use pretty much any of the images on it.  I might recommend starting with the Waveshare Zero image.
